src/server/game/AI/ScriptedAI/ScriptedCreature.cpp

```diff
@@ -517,6 +517,8 @@ void BossAI::_JustDied()
     {
         instance->SetBossState(_bossId, DONE);
         instance->SaveToDB();
+        if (me->GetGUID() == 11121) //Boss Ragnaross
```

Comment:
Sorry, but this is the wrong place for custom content. I think to solve the issue it suffices to call "BossAI::JustDied" in "boss_ragnaros.cpp":
```cpp
void JustDied(Unit* killer) override
{
    BossAI::JustDied(killer);
    me->SetFacingTo(DEATH_ORIENTATION);
}
```